### PR TITLE
Sprint Retrospective for week 5

### PR DESCRIPTION
**Pull request by sanvidpunde**
Sprint Retrospective for week 5 for group 3
Please add or remove points as needed.

### CODE REVIEW

**Review by RahulKrishnaGajula — APPROVED** [review]
Points seem valid. We have done some good work this week.

Some points that could be added are:
1. We need to stop taking on tasks before completing the current task
2. Prepare a common documentation file with guidlines of how to setup and run the system so everyone is on the same page.


Let me know if these points sound reasonable, if so, I'll add them to the Retro and make a new PR

File: Retrospective/week_5_(group3).md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- Need to integrate Interest profiler with scrapped dataset and have a sample output ready before presentation
+- Make Progress in Recommeder system 
+	- Data correction is to be done for column "CAO points" in m_dataset. (more accurate data scrapping is required if needed)
+	- For Collaborative filtering-based systems with LSTM, we need to have it display more than 5 fields in the prefernce output. (CNN is not the most accurate)
+	- Content-based Recommender Systems will be main point of focus for week 5,6,7 as we will be improving on the accuray rate of the output.
+- UI is not scalable as of now, We need to make it dynamically scalable on all the devices by end of next week.
+- Existing Sentiment analysis notebook is being extended to work with the scraped reviews (r_dataset) of TUD campuses.
+
+## Important Points to be taken into account
+- Delete old source code of the website which is not in use from github
```
Comment (themehulprajapati) [inline Retrospective/week_5_(group3).md:13]: Old source code of the website has already been removed. Responsible frontend code is under frontend folder and responsible backend code is under backend folder.

**Review by themehulprajapati — APPROVED** [review]

**Review by RakshithGonsalves — CHANGES_REQUESTED** [review]
Hi, 
Had a look at the retrospective and I completely agree.
There are also some points we need to add.
1. Approach the college more aggressively to get at least the non sensitive data for clubs and societies.

**Comment by sanvidpunde** [comment]
> Hi, Had a look at the retrospective and I completely agree. There are also some points we need to add.
> 
> 1. Approach the college more aggressively to get at least the non sensitive data for clubs and societies.

Added the required changes, Please review and approve. Thanks..

**Review by RakshithGonsalves — APPROVED** [review]
Hi Sanvid, 
Looks good, will let you know if anything needs to be added.

**Pull request merged**